pom.xml

```diff
@@ -104,6 +104,7 @@
         <curator.version>5.6.0</curator.version>
         <zookeeper.version>3.9.1</zookeeper.version>
         <fastjson.version>1.2.83</fastjson.version>
+        <jackson.version>2.13.5</jackson.version>
```

Comment:
The version picked from Spark 3, can change this if we needed

Reply:
It seems that different projects may depend on different versions of Jackson, just like Guava. Should we do better shade them for Amoro and use the shaded classes?

Reply:
Just like what Paimon had do: https://mvnrepository.com/artifact/org.apache.paimon/paimon-shade-jackson-2